resources/plugins/QuantME-CamundaPlugin/quantme/replacement/QuantMETransformator.js

```diff
@@ -54,6 +58,173 @@ export async function startReplacementProcess(xml, currentQRMs, endpointConfig)
     return { status: 'transformed', xml: xml };
   }
 
+  // check for hybrid loop groups
+  const hybridRuntimeGroups = getHybridRuntimeGroups(rootElement, elementRegistry);
+
+  for (let hybridRuntimeGroup of hybridRuntimeGroups) {
+    let groupModeler = await createModelerFromXml(xml);
+    groupModeler.config = endpointConfig;
+    let groupModeling = groupModeler.get('modeling');
+    let groupElementRegistry = groupModeler.get('elementRegistry');
+    const groupDefinitions = groupModeler.getDefinitions();
+    const groupRootElement = getRootProcess(groupDefinitions);
+    console.log(groupElementRegistry,xml);
+
+    let groupElement = groupElementRegistry.get(hybridRuntimeGroup.group.id);
+    let groupX, groupY, groupWidth, groupHeight;
+    groupX= groupElement.x;
+    groupY = groupElement.y;
+    groupWidth = groupElement.width;
+    groupHeight = groupElement.height;
+
+    let elementsInGroup = [];
+    let elementIdsInGroup =[];
+    for (let flowElement of groupRootElement.flowElements) {
+      if (flowElement.$type !== 'bpmn:StartEvent' && flowElement.$type !== 'bpmn:EndEvent') {
+        let flowElementBo = groupElementRegistry.get(flowElement.id);
+
+        // check for each sequence flow if all points are within the group
+        if (flowElementBo.waypoints) {
+          let inside = true;
+          for (let waypoint of flowElementBo.waypoints) {
+            if (!(waypoint.x > groupX && waypoint.x < groupX + groupWidth
+              && waypoint.y > groupY && waypoint.y < groupY + groupHeight)) {
+              inside = false;
+            }
+          }
+          if (inside) {
+            elementsInGroup.push(flowElement);
+            elementIdsInGroup.push(flowElement.id);
+          }
+
+        // check for tasks and gateways if all corners of the element are within the group
+        } else if (flowElementBo.x && flowElementBo.y && flowElementBo.width && flowElementBo.height) {
+          if (flowElementBo.x > groupX && flowElementBo.x < groupX + groupWidth
+            && flowElementBo.x + flowElementBo.width < groupX + groupWidth
+            && flowElementBo.y > groupY && flowElementBo.y < groupY + groupHeight
+            && flowElementBo.y + flowElementBo.height < groupY + groupHeight) {
+            elementsInGroup.push(flowElement);
+            elementIdsInGroup.push(flowElement.id);
+          }
+        }
+      }
+    }
+
+    console.log(elementsInGroup);
```

Comment:
Looks like debugging output while developing? I think it can be removed, but otherwise please add some descriptive text before the elements